projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -273,6 +315,11 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
      * @hidden
      */
     public ngAfterViewInit() {
+        if (this.hasContentTabs) {
+            // in content tabs mode there there is not first tab selected by default
```

Comment:
Some typos in the comment => in content tabs mode no tab is selected by default

Reply:
Addressed.